packages/yew/src/functional/hooks/mod.rs

```diff
@@ -3,29 +3,21 @@ mod use_context;
 mod use_effect;
 mod use_force_update;
 mod use_memo;
-#[cfg_attr(documenting, doc(cfg(any(target_arch = "wasm32", feature = "tokio"))))]
```

Comment:
The requirement of either `wasm32` or `tokio` is removed as now we have `rt_none` and prepared states will compile regardless of whether a runtime is present.